src/main/database.cpp

```diff
@@ -325,8 +305,23 @@ Allocator &Allocator::Get(AttachedDatabase &db) {
 	return Allocator::Get(db.GetDatabase());
 }
 
-void DatabaseInstance::Configure(DBConfig &new_config) {
+void DatabaseInstance::Configure(DBConfig &new_config, const char *database_path) {
```

Comment:
Maybe `new_config` can be const here now?

Reply:
Sadly it can't be, because we do steal things like custom result collector, buffer manager etc from the user-provided config if they are set